.ci/install_firecracker.sh

```diff
@@ -63,10 +63,10 @@ path="/usr/local/bin/kata-runtime"
 
 if [ -f $docker_configuration_file ]; then
 	# Check devicemapper flag
-	check_devicemapper=$(grep -w '"storage-driver": "${driver}"' $docker_configuration_file | wc -l)
+	check_devicemapper=$(grep -w '"storage-driver": "'${driver}'"' $docker_configuration_file | wc -l)
```

Comment:
Thanks @GabyCT 
Can you explain why both `"` and `'` are necessary here?

Reply:
@marcov the docker configuration should looks like
```
 "storage-driver": "devicemapper"
```
however, before like I have it `grep -w '"storage-driver": "${driver}"'` the variable of `driver` was not being replaced and that was causing to the script to fail when you run it multiple times

Reply:
it was doing this the script
```
grep -w '"storage-driver": "${driver}"' /etc/docker/daemon.json
```
but now with this change is replacing the variable correctly
```
grep -w '"storage-driver": "devicemapper"' /etc/docker/daemon.json
```

Reply:
Ahhhhh, now it's clear.
you are concatenating three strings, and the 1st and 3rd are single quoted:
 `"storage-driver": "` + `${driver}` + `"`